Mathlib/Algebra/GroupWithZero/Units/Basic.lean

```diff
@@ -454,10 +453,9 @@ end CommGroupWithZero
 
 section NoncomputableDefs
 
-open scoped Classical
-
 variable {M : Type*} [Nontrivial M]
 
+open Classical in
```

Comment:
```suggestion
open scoped Classical in
```

Reply:
I have used `open Classical in` instead of `open scoped Classical in` as it doesn't make a real difference (if it created a name conflict, one would spot it right away) and is a little bit shorter. If you want, I can change  all over the PR, just tell me.